### DISCUSSION TITLE
Automated Testing

### DISCUSSION THREAD

**Discussion by jrobiii** [mgmeyers/obsidian-kanban#519] [Ideas & Feature Requests]
I am not a test engineer nor an automation guru, but has any thought been given to automated testing?

**Comment by mgmeyers** [comment]
Hi @jrobiii, Yeah, I've been thinking a lot about testing recently. It's something I'd like to add, but I'm currently in grad school, so time is a significant barrier.

**Comment by jrobiii** [comment]
No worries.  I'm not a JS guy, but I can do some research and post thoughts as I get to them.

And thanks for taking the time to respond so quickly.
